sqsjobs/item.go

```diff
@@ -161,36 +169,48 @@ func (i *Item) Nack() error {
 		return nil
 	}
 
-	// requeue message
-	err := i.Options.requeueFn(context.Background(), i)
-	if err != nil {
-		return err
-	}
+	if !i.Options.RetainFailedJobs {
```

Comment:
If we leave this, I'd suggest defaulting it to false. So the default behavior would be like previous, message gets deleted after some retries.

Reply:
It is already defaulted to false.